WMF Framework/UIFont+WMFDynamicType.swift

```diff
@@ -143,10 +143,9 @@ public extension UIFont {
     }
     
     func with(traits: UIFontDescriptor.SymbolicTraits) -> UIFont {
-        guard let descriptor = self.fontDescriptor.withSymbolicTraits(traits) else {
+        guard let descriptor = fontDescriptor.withSymbolicTraits(fontDescriptor.symbolicTraits.union(traits)) else {
             return self
         }
-        
         return UIFont(descriptor: descriptor, size: 0)
```

Comment:
This `with(traits:)` call is used to apply the italic trait to the bold font, but it wasn't working. The issue was not adding the new `traits` to the existing traits, previously only the new traits were applied instead of new + old